Src/BootCamp.Chapter/UI.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class UI
+    {
+        readonly char passwordChar = '*';
+
+        public Credentials GetUserProfile(string action, string message)
+        {
+            Console.Clear();
+            Console.WriteLine(action);
+            if (!string.IsNullOrEmpty(message))
+            {
+                Console.WriteLine(message);
+            }
+            Credentials returnCredential;
+            try
+            {
+                returnCredential = PromptAccount();
+            }
+            catch (ArgumentException)
+            {
+                returnCredential = GetUserProfile(action, "Name cannot be empty!");
+            }
+            return returnCredential;
+        }
+
+        Credentials PromptAccount()
+        {
+            Console.Write("Username: ");
+            string username = Console.ReadLine();
+            Console.Write("Password: ");
+            string unparsedPassword = HiddenReadLine(passwordChar);
+            
+            var utf8 = Encoding.UTF8;
+            var bytes = utf8.GetBytes(unparsedPassword);
+            string password = string.Join("", bytes);
+            return new Credentials(username, password);
+        }   
+
+        string HiddenReadLine(char hidenChar)
```

Comment:
``HiddenReadLine`` is a noun. ``ReadLineHidden`` is a verb. I suggest using that.